llama_index/llms/vertex_utils.py

```diff
@@ -147,7 +147,7 @@ def _parse_message(message: ChatMessage, is_gemini: bool) -> Any:
             convert_chat_message_to_gemini_content,
         )
 
-        return convert_chat_message_to_gemini_content(message)
+        return convert_chat_message_to_gemini_content(message=message, is_history=False)
```

Comment:
could we set default `is_history= true` in order not to change existing behaviour?

Reply:
done